boards/cogip2018-cortex/Makefile.features

```diff
@@ -0,0 +1,15 @@
+# Put defined MCU peripherals here (in alphabetical order)
```

Comment:
Following list is not in alphabetical order

Reply:
Done